cmd/kahoy/apply.go

```diff
@@ -22,6 +22,7 @@ import (
 	managedryrun "github.com/slok/kahoy/internal/resource/manage/dryrun"
 	"github.com/slok/kahoy/internal/resource/manage/kubectl"
 	managekubectl "github.com/slok/kahoy/internal/resource/manage/kubectl"
+	"github.com/slok/kahoy/internal/resource/manage/timeout"
```

Comment:
nit: Use `managetimeout "github.com/slok/kahoy/internal/resource/manage/timeout"`

Reply:
Sure! Whats the reason behind this? Maybe that the packages names although obvious for someone who writes the service could be confused with stdlib packages?

Reply:
Yep!